<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StickyNote</class>
 <widget class="QWidget" name="StickyNote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StickyNote</string>
  </property>
  <property name="windowIcon">
   <iconset resource="stickynote.qrc">
    <normaloff>:/StickyNote/Resources/images/sticky-note.ico</normaloff>:/StickyNote/Resources/images/sticky-note.ico</iconset>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_base" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QWidget" name="widget_buttons" native="true">
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QToolButton" name="tb_align_center">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="stickynote.qrc">
             <normaloff>:/StickyNote/Resources/images/align-center.png</normaloff>:/StickyNote/Resources/images/align-center.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QToolButton" name="tb_Setting">
           <property name="toolTip">
            <string>Settings</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="stickynote.qrc">
             <normaloff>:/StickyNote/Resources/images/settings-red.png</normaloff>:/StickyNote/Resources/images/settings-red.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QToolButton" name="pb_minimize">
           <property name="toolTip">
            <string>Minimize</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="stickynote.qrc">
             <normaloff>:/StickyNote/Resources/images/minimize.png</normaloff>:/StickyNote/Resources/images/minimize.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="tb_StrikeOut">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="stickynote.qrc">
             <normaloff>:/StickyNote/Resources/images/strikethrough2.png</normaloff>:/StickyNote/Resources/images/strikethrough2.png</iconset>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QToolButton" name="tb_align_left">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="stickynote.qrc">
             <normaloff>:/StickyNote/Resources/images/align-left.png</normaloff>:/StickyNote/Resources/images/align-left.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="tb_align_right">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="stickynote.qrc">
             <normaloff>:/StickyNote/Resources/images/align-right.png</normaloff>:/StickyNote/Resources/images/align-right.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="8">
          <widget class="QToolButton" name="pb_close">
           <property name="toolTip">
            <string>Close</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="stickynote.qrc">
             <normaloff>:/StickyNote/Resources/images/close.png</normaloff>:/StickyNote/Resources/images/close.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_pin"/>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="tb_underline">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="stickynote.qrc">
             <normaloff>:/StickyNote/Resources/images/underline.png</normaloff>:/StickyNote/Resources/images/underline.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="8">
          <widget class="QToolButton" name="tb_save">
           <property name="toolTip">
            <string>Save</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="stickynote.qrc">
             <normaloff>:/StickyNote/Resources/images/Save.svg</normaloff>:/StickyNote/Resources/images/Save.svg</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="tb_bold">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="stickynote.qrc">
             <normaloff>:/StickyNote/Resources/images/bold.png</normaloff>:/StickyNote/Resources/images/bold.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3" colspan="3">
          <widget class="QLabel" name="label_title">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Sticky Note</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QToolButton" name="tb_load">
           <property name="toolTip">
            <string>OpenNote</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="stickynote.qrc">
             <normaloff>:/StickyNote/Resources/images/folder-open.svg</normaloff>:/StickyNote/Resources/images/folder-open.svg</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_textEdit" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QTextEdit" name="textEdit">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QWidget" name="widget_labelMessage" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="1">
          <widget class="QLabel" name="label_message">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">
color: rgb(0, 170, 0);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="stickynote.qrc"/>
 </resources>
 <connections/>
</ui>
